src/App.tsx

```diff
@@ -15,48 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'Sort alphabetically',
+  Length = 'Sort by length',
+}
+
+interface SortField {
+  sortField: string | '';
```

Comment:
instead of adding this additional option , you can add it to `enum SortType` as a `DefaultValue = ''`

and here write the next type:
```
interface SortField {
  sortField: SortType;
```